clinica/iotools/converters/genfi_to_bids/genfi_to_bids_utils.py

```diff
@@ -775,21 +802,47 @@ def write_bids(
             metadata["bids_filename"],
             True,
         )
-        if (
-            "dwi" in metadata["bids_filename"]
-            and "Philips" in metadata.manufacturer
-            and dcm2niix_success
-        ):
-            merge_philips_diffusion(
-                to / Path(bids_full_path).with_suffix(".json"),
-                metadata.number_of_parts,
-                metadata.run_num,
+        if dcm2niix_success:
+            scans_filepath = (
+                to
```

Comment:
These changes should not be there I believe. They belong to a previously merged PR.